### PR TITLE
chore: roll Playwright to 1.32.0-alpha-mar-15-2023

### PR DESCRIPTION
**Pull request by mxschmitt**
https://github.com/microsoft/playwright-python/issues/1777

### CODE REVIEW

File: playwright/_impl/_connection.py [hunk 1]
```diff
@@ -271,6 +279,12 @@ def call_on_object_with_known_name(
     ) -> None:
         self._waiting_for_object[guid] = callback
 
+    def start_collecting_call_metadata(self, collector: Any) -> None:
```
Comment (yury-s) [inline playwright/_impl/_connection.py:282]: nit: maybe check it's not yet in the list?

File: playwright/_impl/_connection.py [hunk 2]
```diff
@@ -466,11 +502,15 @@ def _extract_metadata_from_stack(
...
-            stack.append(
+            parsed_frames.append(
                 {
-                    "file": frame.filename,
-                    "line": frame.lineno,
-                    "function": method_name,
+                    "frame": {
+                        "file": frame.filename,
+                        "line": frame.lineno,
+                        "column": 0,
```
Comment (yury-s) [inline playwright/_impl/_connection.py:510]: is there no column in python too?

File: playwright/_impl/_connection.py [hunk 3]
```diff
@@ -480,9 +520,10 @@ def _extract_metadata_from_stack(
...
         api_name = last_internal_api_name
-    if api_name:
-        return {
-            "apiName": api_name,
-            "stack": stack,
-        }
-    return None
+
+    return {
+        "allFrames": [frame["frame"] for frame in parsed_frames],
```
Comment (yury-s) [inline playwright/_impl/_connection.py:525]: Why do we need frameTexts and allFrames?

**Review by yury-s — APPROVED** [review]

File: playwright/_impl/_connection.py (hunk 2 above)
Comment (mxschmitt) [reply playwright/_impl/_connection.py]: ack

**Pull request merged**